src/api/managements/entities/user.entity.ts

```diff
@@ -0,0 +1,33 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+type ManagerForm = {
+  name: string;
+  email: string;
+  verifyToken: string;
+};
+
+@Entity('Managers')
+export class Manager {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ length: 10 })
+  name: string;
+
+  @Column({ unique: true })
+  email: string;
+
+  @Column()
+  password: string;
```

Comment:
password 는 클라이언트에게 응답될 일이 없는 필드이므로 아래처럼 만들어보면 어떨까 해요 select 를 false 를 주면 기본 응답에서는 빠지는 필드가 됩니다 [참고우](https://typeorm.io/entities#column-options)

```suggestion
  @Column({ select: false })
  password: string;
```